### PR TITLE
Change the format of the exported fields, to use '_'

### PR DESCRIPTION
**Pull request by monicasarbu**

### CODE REVIEW

File: beater/lsbeat.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (bt *Lsbeat) listDir(dirFile string, beatname string, init bool) {
 		}
 
 		if f.IsDir() {
-			bt.listDir(dirFile+"/"+f.Name(), beatname, counter)
+			bt.listDir(dirFile+"/"+f.Name(), beatname, false)
```
Comment (urso) [inline beater/lsbeat.go]: ups. Should be `init`, not `false` here.

File: docs/fields.asciidoc [hunk 2]
```diff
@@ -76,25 +76,25 @@ PLEASE UPDATE DOCUMENTATION
 
 
 [float]
-=== modTime
+=== mod_time
```
Comment (urso) [inline docs/fields.asciidoc]: `modtime` might be ok too.

File: docs/fields.asciidoc [hunk 3]
```diff
@@ -76,25 +76,25 @@ PLEASE UPDATE DOCUMENTATION
...
 
 type: date
 
 [float]
-=== filename
+=== file_name
```
Comment (urso) [inline docs/fields.asciidoc]: i'd say stick with `filename` here.

File: docs/fields.asciidoc [hunk 4]
```diff
@@ -76,25 +76,25 @@ PLEASE UPDATE DOCUMENTATION
...
 
 type: text
 
 [float]
-=== fullname
+=== full_name
 
 [float]
-=== isDirectory
+=== is_directory
```
Comment (urso) [inline docs/fields.asciidoc]: `is_directory` or `directory`.

File: beater/lsbeat.go (hunk 1 above)
Comment (monicasarbu) [inline beater/lsbeat.go]: you are right

File: beater/lsbeat.go [hunk 5]
```diff
@@ -75,13 +75,13 @@ func (bt *Lsbeat) listDir(dirFile string, beatname string, init bool) {
...
-			"modTime":     common.Time(t),
-			"filename":    f.Name(),
-			"fullname":    dirFile + "/" + f.Name(),
-			"isDirectory": f.IsDir(),
-			"fileSize":    f.Size(),
+			"@timestamp":   common.Time(time.Now()),
+			"type":         beatname,
+			"mod_time":     common.Time(t),
+			"file_name":    f.Name(),
+			"full_name":    dirFile + "/" + f.Name(),
```
Comment (ruflin) [inline beater/lsbeat.go]: isn't that just the `path` ?

File: beater/lsbeat.go [hunk 6]
```diff
@@ -75,13 +75,13 @@ func (bt *Lsbeat) listDir(dirFile string, beatname string, init bool) {
...
+			"is_directory": f.IsDir(),
```
Comment (ruflin) [inline beater/lsbeat.go]: @monicasarbu No rule exists yet, but we could abbreviate `directory` always by `dir`: https://github.com/elastic/beats/blob/master/metricbeat/module/doc.go#L28 Or are there other abbreviations that could conflict with?

File: beater/lsbeat.go (hunk 5 above)
Comment (monicasarbu) [inline beater/lsbeat.go]: yes, I agree `path` is better. I didn't want to change too much the original names.

File: beater/lsbeat.go (hunk 6 above)
Comment (monicasarbu) [inline beater/lsbeat.go]: Yes, `dir` can also be an option, but I would stick to `directory` in this example beat.

File: beater/lsbeat.go [hunk 7]
```diff
@@ -55,8 +55,8 @@ func (bt *Lsbeat) Run(b *beat.Beat) error {
 		}
 
 		now := time.Now()
-		bt.listDir(bt.config.Path, b.Name, false) // call lsDir
-		bt.lastIndexTime = now                    // mark Timestamp
+		bt.listDir(bt.config.Path, b.Name, true) // call lsDir
```
Comment (urso) [inline beater/lsbeat.go]: keep `false` as last parameter.

**Pull request merged**